extensions/camera/README.md

```diff
@@ -5,7 +5,7 @@ change**
 
 - **Title**: Camera
 - **Identifier**:
-  [https://linz.github.io/stac/__STAC_VERSION__/camera/schema.json]()
```

Comment:
wonder why this was changed?

Reply:
Is that due to Markdown linting? Underscores and asterisks are emphasis indicators in Markdown, so I wonder if it would be better to use something like `%VERSION%` instead? % is used for templating (jinja at least) and I don't think it has a special meaning in Markdown.

`STAC_VERSION` also made me think this was directly related to the STAC spec version, rather than this repo, so maybe the term STAC could be dropped here.